### PR TITLE
Add constraints on invalid password attempts

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

**Review by whitewum — DISMISSED** [review]

File: docs-2.0/7.data-security/1.authentication/1.authentication.md [hunk 1]
```diff
@@ -16,7 +16,13 @@ Local authentication indicates that usernames and passwords are stored locally o
 
 ### Enable local authentication
 
-1. Modify the `nebula-graphd.conf` file (`/usr/local/nebula/etc/` is the default path), set `--enable_authorize=true` and save the modification.
+1. Modify the `nebula-graphd.conf` file (`/usr/local/nebula/etc/` is the default path), set the parameters as followis:
```
Comment (randomJoe211) [inline docs-2.0/7.data-security/1.authentication/1.authentication.md:19]:
```suggestion
1. Modify the `nebula-graphd.conf` file (`/usr/local/nebula/etc/` is the default path) to set the following parameters:
```

File: docs-2.0/7.data-security/1.authentication/1.authentication.md [hunk 2]
```diff
@@ -16,7 +16,13 @@ Local authentication indicates that usernames and passwords are stored locally o
...
+
+  - `--enable_authorize`: Whether to enable authentication, optional values are true or false.
```
Comment (randomJoe211) [inline docs-2.0/7.data-security/1.authentication/1.authentication.md:21]:
```suggestion
  - `--enable_authorize`: Set its value to `true` to enable authentication.
```

Here we are telling the users how to enable authentication, not explaining the parameter.

**Review by randomJoe211 — CHANGES_REQUESTED** [review]

File: docs-2.0/7.data-security/1.authentication/1.authentication.md [hunk 3]
```diff
@@ -16,7 +16,13 @@ Local authentication indicates that usernames and passwords are stored locally o
...
+
+  - `--failed_login_attempts`: The parameter is optional, you need to add this parameter manually. The number of consecutive incorrect passwords allowed by a single Graph service. When the number exceeds the limitation, the account will be locked. If there are multiple Graph services, the allowed number of times is `number of services * failed_login_attempts`.
```
Comment (abby-cyber) [inline docs-2.0/7.data-security/1.authentication/1.authentication.md:23]:
```suggestion
  - `--failed_login_attempts`: This parameter is optional, and you need to add this parameter manually. Specify the attempts of continuously entering incorrect passwords for a single Graph service. When the number exceeds the limitation, your account will be locked. For multiple Graph services, the allowed attempts are `number of services * failed_login_attempts`.
```

File: docs-2.0/7.data-security/1.authentication/1.authentication.md [hunk 4]
```diff
@@ -16,7 +16,13 @@ Local authentication indicates that usernames and passwords are stored locally o
...
+
+  - `--password_lock_time_in_secs`: The parameter is optional, you need to add this parameter manually. Duration of account locking after multiple incorrect passwords is entered. Unit: second.
```
Comment (abby-cyber) [inline docs-2.0/7.data-security/1.authentication/1.authentication.md:25]:
```suggestion
  - `--password_lock_time_in_secs`: This parameter is optional, and you need to add this parameter manually. Specify the time how long your account is locked after multiple incorrect password entries are entered. Unit: second.
```

**Review by foesa-yang — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**